### PR TITLE
Feature/storybook v8

### PR DESCRIPTION
**Pull request by ohp-inmeta**
# Describe your changes

Upgrade storybook to v8
Added addon for a11y testing

## Checklist before requesting a review

- [x] I have performed a self-review and test of my code
- [x] I have added label to the PR (`major`, `minor` or `patch`)
- [ ] If new component: Is story for component created in `stories`-folder?
- [ ] If new component: Is tsx-file import added to `packages/react/index.tsx`?
- [ ] If new component: Is css-file added to `packages/css/index.css`?

### CODE REVIEW

**Comment by ohp-inmeta** [comment]
Storybook v8 💯 
Updated to 8.4.6, then 2 hours later 8.4.7 came out, so updated again :D 

Not sure if this qualifies as a major or minor version, thoughts?

**Comment by aurorascharff** [comment]
If nothing changes for the users of the component library it would not be a major, right?
Remember to set versioning label.

File: .storybook/main.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      options: {
+        mdxPluginOptions: {
+          mdxCompileOptions: {
+            remarkPlugins: [remarkGfm],
+          },
+        },
+      },
+    },
+  ],
+
```
Comment (aurorascharff) [inline .storybook/main.ts:23]: en del tomme linjer som kan fjernes i denne filen, antar det er pga autogenerert config?

**Review by aurorascharff — APPROVED** [review]
Det var jo faktisk ikke så mange endringer! I storybook 6-7 var det veldig mange APIer som måtte endres. Godt jobba :)

File: .storybook/main.ts (hunk 1 above)
Comment (ohp-inmeta) [reply .storybook/main.ts:23]: Ja, det, pluss at jeg liker å ha litt rom mellom parametere, lettere å lese :P

**Pull request merged**